src/infra/datasets/postgres/src/repos/postgres_dataset_entry_repository.rs

```diff
@@ -101,7 +104,7 @@ impl DatasetEntryRepository for PostgresDatasetEntryRepository {
                     dataset_name as name,
                     created_at   as "created_at: _"
                 FROM dataset_entries
-                ORDER BY dataset_name ASC
+                ORDER BY created_at ASC
```

Comment:
Plz explain this change

Reply:
Discussed it in Slack, my argument was about potentially having new entry in while pages are returned -- but eventually agreed that we could remove that change

Removed